docs/articles/documentation/guides/basic-guides/debug.md

```diff
@@ -1,43 +1,35 @@
 ---
 layout: docs
-title: Debugging
-permalink: /documentation/test-api/debugging.html
-checked: true
+title: Debug
+permalink: /documentation/guides/basic-guides/debug.html
 ---
-# Debugging
+# Debug
 
 TestCafe allows you to debug server-side test code and test behavior on the client.
 
-* [Debugging Test Code](#debugging-test-code)
+* [Debug Test Code](#debug-test-code)
 * [Client-Side Debugging](#client-side-debugging)
 * [Options Useful for Debugging](#options-useful-for-debugging)
 
-## Debugging Test Code
+## Debug Test Code
 
 You can debug test code in Chrome Developer Tools and popular IDEs. See the following recipes for details.
```

Comment:
recipes?

Reply:
"Recipes" or, collectively, "Cookbook" is a common term for documents that show specific scenarios in depth.

UI frameworks like `react` or `vue` have such sections:
https://reactjs.org/docs/testing-recipes.html
https://vuejs.org/v2/cookbook/

Most tools I use in my work, like site generators, also do:
https://www.gatsbyjs.org/docs/recipes/